WcaOnRails/app/models/person.rb

```diff
@@ -11,6 +11,13 @@ class Person < ActiveRecord::Base
 
   scope :current, -> { where(subId: 1) }
 
+  scope :in_region, lambda { |region|
```

Comment:
Just curious about your thoughts here. In my experience, I've seen brackets `{}` used for one-liners, and `do...end` used for multiliners. I'm not sure why that is, but I think that's the convention we've been following throughout our codebase.

Also, one clever thing @Baiqiang showed me is that all our [continent ids](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/db/seeds/continents.seeds.rb) start with an underscore. I've always wondered why that is, but @Baiqiang believes it's so that code dealing with regions can distinguish between countries and continents (continents always start with a `_`). I can't decide if it's gross, or clever, but you could make use of that here.